config/when/when.go

```diff
@@ -165,6 +167,25 @@ func (w *When) validateExists() error {
 	return newCondFailErrorf("no required file existed: %s", w.Exists)
 }
 
+func (w *When) validateNotExists() error {
+	if len(w.NotExists) == 0 {
+		return newUnspecifiedError("not-exists")
+	}
+
+	for _, f := range w.NotExists {
+		if _, err := os.Stat(f); err != nil {
+			if os.IsNotExist(err) {
+				continue
```

Comment:
There's actually a bit of nuance with how this works, but with the current setup, you can get either "any" or "all" behavior. 

https://rliebz.github.io/tusk/spec/#when-anyall-logic

Here's a simple example you can test that out with locally:

```yml
tasks:

  exists:
    usage: Print which files exists
    run:
       - when:
          exists:
            - tusk.yml
            - fake.txt
        command: echo "At least one exists"
      - when:
          - exists: tusk.yml
          - exists: fake.txt
        command: echo "They both exist"
```

For the current PR, this just means updating the behavior/test cases/docs to match what's being done for the other clauses, which is that within a single `when` clause, the behavior should be "any" rather than "all". 

I'll update that documentation as well to make things a bit clearer.

Reply:
This line should then be `return nil`

Reply:
I've changed the docs. Should I update that too?

Reply:
Yeah, that'd be great

Reply:
Awesome! I'm making the changes and doing some more tests to understand the behavior. It's quite strange right now tbh but I think I got it